src/Shouldly/App_Packages/JetBrainsAnnotations/JetBrainsAnnotations.cs

```diff
@@ -1,5 +1,8 @@
-﻿using System;
+﻿// <auto-generated/>
```

Comment:
This is recognized by the compiler. It causes the file to be excluded from analysis.